src/NuGet.Indexing/ResponseFormatter.cs

```diff
@@ -340,6 +342,7 @@ private static void WriteDataV2(JsonWriter jsonWriter, NuGetIndexSearcher search
                 WriteDocumentValue(jsonWriter, "ReleaseNotes", document, "ReleaseNotes");
                 WriteDocumentValue(jsonWriter, "ProjectUrl", document, "ProjectUrl");
                 WriteDocumentValue(jsonWriter, "IconUrl", document, "IconUrl");
+                WriteProperty(jsonWriter, "IsVerified", isVerified);
```

Comment:
For v2 this needs to be in `PackageRegistration` object, `WriteRegistrationV2` above. See the OneNote spec for the return result structure. Also the key should be "Verified" for V2 search result on the registration object.